backend/app/services/prompt_draft_service.py

```
@@ -0,0 +1,558 @@
+        skill_name = str(arguments.get("skill_name") or "").strip()
+        if skill_name:
+            return f"尝试加载技能 {skill_name}"
+        return "尝试加载技能"
+    return f"尝试调用工具 {name}"
+
+
+def _summarize_tool_result(message: dict[str, Any]) -> str | None:
+    tool_name = str(message.get("name") or "").strip()
+    content = str(message.get("content") or "").strip()
+    if tool_name == "load_skill":
+        match = re.search(r"Skill '([^']+)' has been loaded", content)
```

Ruff (0.15.7)

[error] Undefined name `re`

(F821)